quickstart.py

```diff
@@ -46,18 +51,25 @@ def main():
     "config JSON file.")
   group.add_argument('--configJSON', metavar = "JSON", help = "Literal JSON to "
     "load configuration from rather than a configuration file.")
-  group.add_argument('--stop', action = "store_true", help = "If specified, "
+  parser.add_argument('--stop', action = "store_true", help = "If specified, "
     "this script will stop all servers rather than starting them. Note that "
     "this will not necessarily work if quickstart was not used to start the "
     "servers.")
+  parser.add_argument('--foreground', '-F', action = "store_true",
```

Comment:
Docker needs quickstart to run in the foreground so that the container continues to run.